packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbar.md

```diff
@@ -0,0 +1,399 @@
+  }
+}
+
+```
+## Data toolbar toggle groups
+```js
+import React from 'react';
+import { DataToolbar , DataToolbarItem, DataToolbarContent, DataToolbarToggleGroup } from '@patternfly/react-core/dist/esm/experimental';
+import { Button, InputGroup, Select, SelectOption } from '@patternfly/react-core';
+import { TextInput, SearchIcon, FilterIcon } from '@patternfly/react-icons'
+
+class DataToolbarToggleGroupExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputValue: "",
+      firstIsExpanded: false,
+      firstSelected: null,
+      secondIsExpanded: false,
+      secondSelected: null
+    };
+    
+    this.firstOptions = [
+      { value: 'Status', disabled: false, isPlaceholder: true },
+      { value: 'New', disabled: false },
+      { value: 'Pending', disabled: false },
+      { value: 'Running', disabled: false },
+      { value: 'Cancelled', disabled: false },
+    ];
+    
+    this.secondOptions = [
+      { value: 'Risk', disabled: false, isPlaceholder: true },
+      { value: 'Low', disabled: false },
+      { value: 'Medium', disabled: false },
+      { value: 'High', disabled: false },
+    ];
+    
+    this.onInputChange = (newValue) => {
+         this.setState({inputValue: newValue});
+        };
+    
+    this.onFirstToggle = isExpanded => {
+      this.setState({
+        firstIsExpanded: isExpanded
+      });
+    };
+    
+    this.onFirstSelect = (event, selection, isPlaceholder) => {
+      if (isPlaceholder) this.clearFirstSelection();
+      this.setState({
+        firstSelected: selection,
+        firstIsExpanded: false
+      });
+    };
+    
+    this.clearFirstSelection = () => {
+      this.setState({
+        firstSelected: null,
+        firstIsExpanded: false
+      });
+    };
+    
+    this.onSecondToggle = isExpanded => {
+      this.setState({
+        secondIsExpanded: isExpanded
+      });
+    };
+    
+    this.onSecondSelect = (event, selection, isPlaceholder) => {
+      if (isPlaceholder) this.clearSecondSelection();
+      this.setState({
+        secondSelected: selection,
+        secondIsExpanded: false
+      });
+    };
+    
+    this.clearSecondSelection = () => {
+      this.setState({
+        secondSelected: null,
+        secondIsExpanded: false
+      });
+    };
+  }
+  
+  render() {
+    const { inputValue, firstIsExpanded, firstSelected, secondIsExpanded, secondSelected } = this.state;
+
+    const toggleGroupItems = <React.Fragment>
+      <DataToolbarItem>
+        <InputGroup>
+          <TextInput name="textInput2" id="textInput2" type="search" aria-label="search input example" onChange={this.onInputChange} value={inputValue}/>
+          <Button variant={ButtonVariant.tertiary} aria-label="search button for search input">
+            <SearchIcon />
+          </Button>
+        </InputGroup>
+      </DataToolbarItem>
+      <DataToolbarItem spacers={[{spacerSize: 'none'}]}>
```

Comment:
In this case, the filters should be a nested `DataToolbarGroup`, as `__group`s are nestable.

```
<DataToolbarToggleGroup>
  <DataToolbarItem>
    <InputGroup />
  </DataToolbarItem>

  <DataToolbarGroup variant="filter-group">
    <DataToolbarItem>
      <Select />
    </DataToolbarItem>
    <DataToolbarItem>
      <Select />
    </DataToolbarItem>
  </DataToolbarGroup>

  <DataToolbarItem>
    <Stuff />
  </DataToolbarItem>

 </DataToolbarToggleGroup>
```

Reply:
they are nestable, but then they will not take up 100% width in the expandable content like they do in the core example... they will still be grouped like a filter-group in the expandable content...